### DISCUSSION TITLE
Is there any way to disable hardware acceleration for Avalonia App?

### DISCUSSION THREAD

**Discussion by MrReny** [AvaloniaUI/Avalonia#11796] [Q&A] [answered]
There is some strange behaviours with recent gen Intel processors graphics on some old fork of linux. I wonder to try disable hardware acceleration but was unable find out how.

**Comment by kekekeks** [comment, marked as answer]
add `.With(new X11PlatformOptions { UseGpu = false})` to your AppBuilder
